src/core_plugins/tagcloud/public/__tests__/tag_cloud.js

```diff
@@ -440,9 +440,12 @@ describe('tag cloud tests', function () {
 
 
     const centered = (largest[1] === 0 && largest[2] === 0);
+    const halfWidth = debugInfo.size[0] / 2;
```

Comment:
I know this wasn't introduced by this PR, but why are we using an Array for debugInfo.size and the items in debugInfo.positions as opposed to an Object with named properties? Having to know the indexes of the properties makes this kinda hard to follow.